### DISCUSSION TITLE
Tracking Edits by UniqueID

### DISCUSSION THREAD

**Discussion by codepeach** [ueberdosis/tiptap#5694] [General Questions]
I'm using the UniqueID extension for every time a new paragraph is generated. For specific reasons,  I'm saving each paragraph individually in my database and loading them at run time.

Is there a way I can get the information of which element was edited based on UniqueID?

**Comment by nperez0111** [comment]
Sorry, this isn't totally clear.

The unique id extension is mostly about assigning new ids to content, preserving the id if it already existed. So, I would say, no the unique id does not tell you which paragraph would have changed.

Your best bet is to just listen to update events and store your content back like that. If you want to save on DB calls or something, on the client you can compare what was loaded and what is being sent back to the DB to store back only the paragraphs that actually changed
